src/aave-v2/lens/RatesLens.sol

```diff
@@ -142,13 +151,13 @@ abstract contract RatesLens is UsersLens {
                 .onPool;
 
                 if (firstPoolSupplierBalance > 0) {
-                    uint256 matchedP2P = Math.min(
+                    withdrawnFromPool = Math.min(
                         firstPoolSupplierBalance.rayMul(indexes.poolSupplyIndex),
                         _amount
                     );
 
-                    borrowBalance.inP2P += matchedP2P.rayDiv(indexes.p2pBorrowIndex);
-                    _amount -= matchedP2P;
+                    borrowBalance.inP2P += withdrawnFromPool.rayDiv(indexes.p2pBorrowIndex);
+                    _amount -= withdrawnFromPool;
```

Comment:
We can harmonize the way we compute this, I liked the way it was done in `getNextUserSupplyRatePerYear`